### PR TITLE
Devops 121 docker dev

### PR DESCRIPTION
**Pull request by mthambipillai**
Notes:
- The Dockerfile is actually only for the tracker
- I had to remove the host param when creating the WebSocket server so that it would be accessible from outside the Docker container

### CODE REVIEW

File: .travis_scripts/deploy_docker.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+if [ $1 = "staging" ]; then
+    docker push $OWNER/$IMAGE_NAME:$TAG
+elif [ $1 = "production" ]; then
+    docker build -t $OWNER/$IMAGE_NAME:$TAG .
+    #docker tag $OWNER/$IMAGE_NAME:$TAG $OWNER/$IMAGE_NAME:$TRAVIS_TAG
```
Comment (mirpo) [inline .travis_scripts/deploy_docker.sh]: Please remove commented lines.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Use official node runtime as base image
+FROM node:8.11.4-alpine
```
Comment (mirpo) [inline Dockerfile]: We use node 10

File: bin/tracker.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const util = require('util')
 const { startTracker } = require('../src/composition')
 
 const port = process.argv[2] || 30300
-const ip = process.argv[3] || '127.0.0.1'
+const ip = process.argv[3]
```
Comment (mirpo) [inline bin/tracker.js:8]: We use it ./bin/ scripts, they might fail now, better run them to recheck

**Review by mirpo — CHANGES_REQUESTED** [review]

File: bin/tracker.js (hunk 3 above)
Comment (mthambipillai) [reply bin/tracker.js:8]: Ran them locally, none fails

**Review by mirpo — APPROVED** [review]

**Comment by mirpo** [comment]
LGFM!

**Pull request merged**